db/students.ts

```diff
@@ -10,6 +12,32 @@ const insert = async (email: string): Promise<void> => {
   }
 };
 
+const getAllCreatedBetween = async (from: Date, to: Date): Promise<Student[]> => {
+  try {
+    return db.from(studentsTable)
+    .whereBetween('createdAt', [from, to]);
+  } catch (err) {
+    console.error('Erreur lors de la récuperation des étudiants', err);
+    throw new Error('Erreur lors de la récuperation des étudiants');
+  }
+};
+
+const updateById = async (id: string, student: Partial<Student>): Promise<void> => {
```

Comment:
I used `Partial<Student` here but we may use a more restricted object if you find it dangerous.